tasks/puppet_runonce.json

```
@@ -4,6 +4,11 @@
     "noop": {
       "type": "Optional[Boolean]",
       "description": "If true, run Puppet in no-op mode"
+    },
+    "in_progress_timeout": {
+      "type": "Integer",
```

We should prohibit negative values, and probably even 0:
```suggestion
      "type": "Integer[1]",
```
(Integer[firstvalue,secondvalue] sets an allowed range of values. the first value is the lowest allowed value. We don't need to set a second value. If we do it, it's the upper value limit)